pastis/matrix_generation/matrix_building_numerical.py

```diff
@@ -734,6 +817,23 @@ def calculate_ref_image(self, save_coro_floor=True, save_psfs=True):
                                                                                                                save_psfs=save_psfs,
                                                                                                                outpath=self.overall_dir)
 
+class MatrixIntensityRST(PastisMatrixIntensities):
+    instrument = 'RST'
+
+    def __int__(self, initial_path='', savepsfs=True, saveopds=True):
+        super().__init__(design=None, savepsfs=savepsfs, saveopds=saveopds)
+
+    def setup_one_pair_function(self):
+        self.calculate_matrix_pair = functools.partial(_rst_matrix_one_pair, self.norm, self.wfe_aber, self.resDir,
+                                                       self.savepsfs, self.saveopds)
+
+    def calculate_ref_image(self, save_coro_floor=False, save_psfs=False, outpath=''):
+        self.contrast_floor, self.norm, self.coro_simulator = calculate_unaberrated_contrast_and_normalization('RST',
+                                                                                                               return_coro_simulator=True,
+                                                                                                               save_coro_floor=save_coro_floor,
+                                                                                                               save_psfs=save_psfs,
+                                                                                                               outpath=outpath)
```

Comment:
This would be better with `outpath=self.overall_dir`, like in `MatrixIntensityLuvoirA()`, and then you can delete the `outpath` input parameter. This is only for the one single ref image (well, actually a pair of direct + coro), so it's fine (and actually advantageous) to save it directly in the overall output folder of a simulation run.